include/utils/polygon.h

```diff
@@ -299,6 +299,19 @@ class ConstPolygonRef
         return res == 1;
     }
 
+    bool inside(const auto& polygon) const
+    {
+        for (const auto& point : *path)
+        {
+            const auto res = ClipperLib::PointInPolygon(point, *polygon.path);
+            if (res < 1)
```

Comment:
I think either you should change "res" to a more descriptive name, or do this. in one line.

```suggestion
            if (!ClipperLib::PointInPolygon(point, *polygon.path))
```